src/services/process-service.ts

```diff
@@ -35,7 +36,11 @@ export default class ProcessService {
       const pid = this.getPid()
       fs.unlinkSync(ProcessService.PID_PATH)
 
-      process.kill(pid, 'SIGINT')
+      if (os.platform() === 'win32') {
+        childProcess.exec('taskkill /pid ' + pid + ' /T /F')
+      } else {
+        process.kill(pid, 'SIGHUP')
```

Comment:
You changed from using SIGINT to SIGHUP on non-windows?

Reply:
I think this might be a comment on an old commit 😄. Ended up splitting out `win32` as a separate step and non-windows on the existing `SIGHUP`.